build/infrastructure/main/func-functionhost.tf

```
@@ -57,8 +57,8 @@ module "func_functionhost" {
     CHARGE_COMMAND_RECEIVED_SUBSCRIPTION_NAME                       = "sbtsub-charges-charge-command-received"
     CHARGE_COMMAND_REJECTED_SUBSCRIPTION_NAME                       = "sbtsub-charges-charge-command-rejected"
     CHARGE_PRICE_COMMAND_RECEIVED_SUBSCRIPTION_NAME                 = "sbtsub-charges-charge-price-command-received"
-    CHARGE_PRICE_REJECTED_SUBSCRIPTION_NAME                         = "sbtsub-charges-charge-price-rejected"
-    CHARGE_PRICE_CONFIRMED_SUBSCRIPTION_NAME                        = "sbtsub-charges-charge-price-confirmed"
+    CHARGE_PRICE_OPERATIONS_REJECTED_SUBSCRIPTION_NAME              = "sbtsub-charges-charge-price-rejected"
+    CHARGE_PRICE_OPERATIONS_CONFIRMED_SUBSCRIPTION_NAME             = "sbtsub-charges-charge-price-confirmed"
```

Should we change the subscription names as well?
```suggestion
    CHARGE_PRICE_OPERATIONS_REJECTED_SUBSCRIPTION_NAME              = "sbtsub-charges-charge-price-operations-rejected"
    CHARGE_PRICE_OPERATIONS_CONFIRMED_SUBSCRIPTION_NAME             = "sbtsub-charges-charge-price-operations-confirmed"
```